.lighthouse/jenkins-x/pullrequest.yaml

```diff
@@ -13,17 +13,13 @@ spec:
         stepTemplate:
           image: uses:jenkins-x/jx3-pipeline-catalog/tasks/go-plugin/pullrequest.yaml@versionStream
           name: ""
-          resources:
-            requests:
-              cpu: 400m
-              memory: 600Mi
+          resources: {}
```

Comment:
Breaking change in tekton 0.28+, it now takes the sum of resources in the steps, instead of max.